rocky/tools/forms/ooi_form.py

```diff
@@ -197,3 +197,7 @@ class OOITypeMultiCheckboxForm(BaseRockyForm):
         choices=OOI_TYPE_CHOICES,
         widget=forms.CheckboxSelectMultiple,
     )
+
+
+class OOISearchForm(BaseRockyForm):
+    search = forms.CharField(label=_("Search"), required=False, max_length=256)
```

Comment:
Maybe can also set the help_text argument and explain 'how' this searches.
```suggestion
    search = forms.CharField(label=_("Search"), required=False, max_length=256  help_text="Object ID contains")
```